plugin-gradle/README.md

```diff
@@ -418,6 +418,8 @@ spotless {
       .editorConfigOverride(
         mapOf(
           "indent_size" to 2,
+          // intellij_idea is the default style we preset in Spotless, you can override it referring to https://pinterest.github.io/ktlint/latest/rules/code-styles.
```

Comment:
```suggestion
          // intellij_idea is the default style we preset in Spotless, you can override it referring to https://pinterest.github.io/ktlint/latest/rules/code-styles.
          //this is the example of overriding to `android_studio` style.
```

Reply:
One more thing, a blank should be asserted after `//`.